package.nls.json

```diff
@@ -24,5 +24,13 @@
   "configuration.java.dependency.autoRefresh": "Synchronize Java Projects explorer with changes",
   "configuration.java.dependency.refreshDelay": "The delay time (ms) the auto refresh is invoked when changes are detected",
   "configuration.java.dependency.packagePresentation": "Package presentation mode: flat or hierarchical",
-  "configuration.java.project.exportJar.targetPath": "The default output path of export jar"
+  "configuration.java.project.exportJar.targetPath": "The default output path of export jar.",
+  "taskDefinitions.java.project.exportJar.label": "The label of export jar task.",
+  "taskDefinitions.java.project.exportJar.elements": "The content list of the exported jar.",
+  "taskDefinitions.java.project.exportJar.mainClass": "The main class in the manifest of the exported jar.",
+  "taskDefinitions.java.project.exportJar.targetPath": "The output path of the exported jar. This will override configuration.java.project.exportJar.targetPath.",
+  "taskDefinitions.java.project.exportJar.compileOutput": "All of the output folders in the runtime scope.",
```

Comment:
@jdneo What do you think of the wording here, including `output folders` and `dependencies` ? Are them the proper ones? Or could you please give some suggestions?